src/components/IndexLiquidityTab/IndexLiquiditySimulateDataTableRow.tsx

```diff
@@ -0,0 +1,191 @@
+      } 
+    })
+    const findBestTrade = () => {
+      let sorted = [...exchangeMaxTrades].sort((a,b) => a && b ? b - a : 0)
+      const indexOfBestTrade = exchangeMaxTrades.indexOf(sorted[0])
+      setIsLoading(false)
+      return resultsFromExchanges![indexOfBestTrade]
+    }
+    const bestTrade = findBestTrade()
+    setBestExchange(bestTrade!.exchange)
+    setMaxTrade(bestTrade!.response)
+  }
+  const renderDataTableRow = (component: IndexComponent | undefined) => {
+    if (!component) return null
+    const maxTradeToken =
+      maxTrade!.mul(PRICE_DECIMALS).div(tenPowDecimals).toNumber() /
+      PRICE_DECIMALS
+    const maxTradeUSD =
+      tokenPrice.mul(maxTrade!).div(tenPowDecimals).toNumber() /
+      PRICE_DECIMALS
+    const percentageChange = parseFloat(`${parseFloat(target) - parseFloat(component.percentOfSet)}`).toFixed(2);
+    const dollarChange = parseFloat(`${parseFloat(percentageChange) * 0.01 * selectedIndexMarketCap}`).toFixed(2);
+    const numberOfTrade = Math.abs(Math.round(parseFloat(`${ parseFloat(dollarChange) / maxTradeUSD }`)));
+    const estimatedCost = parseFloat(props.gasCost) * numberOfTrade * 1.3
+    console.log('no. of traders',numberOfTrade );
+    
+
+    return (
+      <>
+        <TableData>{component.symbol}</TableData>
+        <TableDataRightAlign>{component.percentOfSet}</TableDataRightAlign>
+        <TableDataRightAlign>
+          <TextField
+              value={target}
+              onChange={onTarget}
+              onBlur={(e) => findMaxTrade(component)}
+              style={textFieldStyles}
+              inputProps={{
+                autoComplete: 'new-password', // disable autocomplete and autofill
+              }}
+            />
+        </TableDataRightAlign>
+        <TableDataRightAlign>{percentageChange}</TableDataRightAlign>
+        <TableDataRightAlign>{dollarChange}</TableDataRightAlign>
+        <TableDataRightAlign>
+          <TextField
+              value={allowedSlippage}
+              onChange={onSlippageChange}
+              onBlur={(e) => findMaxTrade(component)}
+              style={textFieldStyles}
+              inputProps={{
+                autoComplete: 'new-password', // disable autocomplete and autofill
+              }}
+            />
+        </TableDataRightAlign>
+        <TableDataRightAlign>{isLoading ? <CircularProgress /> : bestExchange}</TableDataRightAlign>
+        <TableDataRightAlign>{isLoading ? <CircularProgress /> : tradeError ? 'error' : formatDisplay(maxTradeToken)}</TableDataRightAlign>
+        <TableDataRightAlign>{isLoading ? <CircularProgress /> : tradeError ? 'error' : formatUSD(maxTradeUSD)}</TableDataRightAlign> 
+        <TableDataRightAlign>{numberOfTrade}</TableDataRightAlign>
+        <TableDataRightAlign>{formatUSD(estimatedCost)}</TableDataRightAlign>
+
+      </>
+    )
+  }
+  if (props.component) {
+    return renderDataTableRow(props.component)
+  }
+  return null
```

Comment:
I forgot to include this when I broke up the Epic into pieces but please add the Total row like in this mockup. And thanks for completing the epic this looks great.

![image](https://user-images.githubusercontent.com/32803520/144450244-2f03a9e6-8bc4-4604-a65c-bcf24bb066fc.jpeg)

Reply:
Sure, I'll add the total row and the turn red if not equal to 100% feature.